packages/snap-controller/src/Search/SearchController.ts

```diff
@@ -759,7 +788,17 @@ function getSearchAddtocartSchemaData({
 	};
 }
 
-function getSearchSchemaData({ params, store, results }: { params: SearchRequestModel; store: SearchStore; results?: Product[] }): SearchSchemaData {
+function getSearchSchemaData({
+	params,
+	store,
+	results,
+	response,
+}: {
+	params: SearchRequestModel;
+	store: SearchStore;
```

Comment:
Does it make sense to use the store values as fallback values here? If not, it seems we shouldn't really need the store.